plugins/modules/icinga2_host.py

```diff
@@ -51,9 +51,9 @@
       - If the O(url_username) parameter is not specified, the O(url_password) parameter will not be used.
   force_basic_auth:
     description:
-      - Httplib2, the library used by the uri module only sends authentication information when a webservice responds to an
-        initial request with a 401 status. Since some basic auth services do not properly send a 401, logins will fail. This
-        option forces the sending of the Basic authentication header upon initial request.
+      - C(httplib2), the library used by the URI module only sends authentication information when a webservice responds to
```

Comment:
I'm not sure whether the `ansible.builtin.uri` module was meant here (which isn't really right), or the `ansible.module_utils.urls` module utils, but the current version definitely isn't right. How about:
```suggestion
      - C(httplib2), the library used by Ansible's HTTP request code only sends authentication information when a webservice responds to
```